meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -239,6 +239,7 @@ do_install() {
         install-bin \
         install-identity-scripts \
         install-inventory-scripts \
+        ${@bb.utils.contains('MENDER_FEATURES_ENABLE', 'mender-inventory-geo-script', 'install-inventory-geo-script', '', d)} \
```

Comment:
As we are discussing at https://github.com/mendersoftware/mender/pull/651/files#r515787653 the target is likely to be called `install-network-inventory-scripts`; therefore the feature here should also align with "network" rather than the specific script (this allows for the future to have more scripts in this category).